[PATCH] fix version number

Somehow the 0.9.33.1 tarball thinks it is 0.9.33.2-git, as pointed out
by Mike Frysinger.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 Rules.mak |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: uClibc-0.9.33.1/Rules.mak
===================================================================
--- uClibc-0.9.33.1.orig/Rules.mak
+++ uClibc-0.9.33.1/Rules.mak
@@ -105,7 +105,7 @@
 MAJOR_VERSION := 0
 MINOR_VERSION := 9
 SUBLEVEL      := 33
-EXTRAVERSION  :=.2-git
+EXTRAVERSION  :=.1
 VERSION       := $(MAJOR_VERSION).$(MINOR_VERSION).$(SUBLEVEL)
 ABI_VERSION   := $(MAJOR_VERSION)
 ifneq ($(EXTRAVERSION),)
